unbox.sh

```diff
@@ -4,4 +4,7 @@ set -e
 
 mv evm/box/* .
 rm -rf evm
-rm box-img*
\ No newline at end of file
+IMAGE=box-img-lg.png
```

Comment:
what about `box-img-sm.png`?

Reply:
This is just a shortcut. If one exists, we can assume they both do.